crates/executor/src/types.rs

```diff
@@ -174,19 +174,11 @@ pub struct ExecutionResources {
     pub segment_arena_builtin: usize,
 }
 
-impl TryFrom<blockifier::execution::call_info::CallInfo> for FunctionInvocation {
-    type Error = blockifier::transaction::errors::TransactionExecutionError;
-
-    fn try_from(
-        call_info: blockifier::execution::call_info::CallInfo,
-    ) -> Result<Self, Self::Error> {
+impl From<blockifier::execution::call_info::CallInfo> for FunctionInvocation {
```

Comment:
Nice catch.

Reply:
Found it when I deleted the dodgy looking `From<BlockifierExecutionError> for TxExError` (since that is also what we are mapping in simulate.